docs/STORAGE.md

```diff
@@ -17,7 +17,12 @@ The default implementation of the `StorageManagerType` and `StorageType` protoco
 
 When clients of this class request a `StorageType`, `CoreDataManager` will return an `NSManagedObjectContext`. 
 
-When `CoreDataManager` is requested a  `viewContext`, it will provide  the persistent container’s `viewContext` . When it is requested a `newDerivedStorage` it will return a new child context with  a private dispatch queue.
+When `CoreDataManager` is requested a  `viewContext`, it will provide  the persistent container’s `viewContext`. `viewContext` should only be used for reading and not writing.
```

Comment:
```suggestion
When `CoreDataManager` is requested a  `viewContext`, it will provide the persistent container’s `viewContext`. `viewContext` should only be used for reading and not writing.
```